src/libstd/time/duration.rs

```diff
@@ -188,6 +188,13 @@ impl DivAssign<u32> for Duration {
     }
 }
 
+#[stable(feature = "duration_default", since = "1.9.0")]
+impl Default for Duration {
+    fn default() -> Duration {
+        Duration::new(0, 0)
```

Comment:
I don't have strong feelings about this, we can definitely remove this if this isn't wanted. Two upsides: Using 0 seems to have some precedence (integers, floats), even though it's not clear where you'll use them. Implementing `Default` for more types means that you can `#[derive]` it in more cases.